local-tests/chainrunner/chain.py

```diff
@@ -212,6 +212,7 @@ def wait_for_finalization(self, old_finalized, nodes=None, timeout=600, finalize
         deadline = time.time() + timeout
         while self.get_highest_finalized(nodes) <= old_finalized + finalized_delta:
             time.sleep(5)
+            print('wait_for_finalization: the highest finalized is ', self.get_highest_finalized(nodes))
```

Comment:
use logging?

Reply:
Yes but in other PRs please - this python test framework is a decent tool but it needs a slightly revamp, so we can tackle that problem there.